src/plots/VolumeVTK9/PyVolumeAttributes.C

```diff
@@ -3456,44 +3456,50 @@ PyVolumeAttributes_getattr(PyObject *self, char *name)
     if(strcmp(name, "materialProperties") == 0)
         return VolumeAttributes_GetMaterialProperties(self, NULL);
 
-    // Try and handle legacy fields
+#include <visit-config.h>
 
-#define DEPRECATED_MESSAGE(type) \
-    PyErr_WarnFormat(NULL, 3, "%s is no longer a valid Volume attribute.\n" \
+#if VISIT_OBSOLETE_AT_VERSION(3,5,0)
+#error This code is obsolete in this verison of VisIt and should be removed.
+#else
+    // Try and handle legacy fields
+#define DEPRECATED_MESSAGE(type, msg) \
+    PyErr_WarnFormat(NULL, 1, "%s is no longer a valid Volume attribute.\n" \
                     "It's value is being ignored, " \
-                    "please remove it from your script.\n", type);
-
+                    "please remove it from your script.\n", \
+                    "Try using %s instead.\n", type, msg);
 #define NAME_CHANGE_MESSAGE(old, new) \
-    PyErr_WarnFormat(NULL, 3, "%s is no longer a valid Volume attribute.\n" \
+    PyErr_WarnFormat(NULL, 1, "%s is no longer a valid Volume attribute.\n" \
```

Comment:
Is the number `1` here which debug level this message is associated with. Good idea to change to `1` if so, That said, doesn't that also mean that nobody sees these warnings if they do not run with `-debug X`?

Reply:
The number is related to python stack levels. I found that the '3' printed weird extra stuff when I was running cli directly.  '1' cleared up the weirdness.

However, still seeing some weird extra printed stuff with '1' when running the test suite, but not as bad.
Here's an example from the test suite.
Don't know why the `#---------------` line is there.
```
visit_test_main.py:90: RuntimeWarning: RayCasting is no longer a valid Volume attribute.
It's name has been changed to Composite, please update your script.

  # ----------------------------------------------------------------------------
```